src/main/java/org/jenkinsci/test/acceptance/docker/fixtures/SshdContainer.java

```diff
@@ -18,7 +18,7 @@
  *
  * @author Kohsuke Kawaguchi
  */
-@DockerFixture(id = "sshd", ports = 22, bindIp = "127.0.0.5")
+@DockerFixture(id = "sshd", ports = 22)
```

Comment:
there was no requirement that I could see for this to be bound to a specific IP address...